src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_Basics/Task4.java

```diff
@@ -0,0 +1,16 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_Basics;
+
+import java.util.Scanner;
+
+public class Task4 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter a last number please:");
```

Comment:
"a last number" is a bit ambiguous, this could be changed to something like:
Enter the last number up to which the sum will be calculated

Reply:
Done.